src/main/java/net/vassbo/vanillaemc/data/EMCValues.java

```diff
@@ -65,230 +70,230 @@ public static void init() {
-        EMC_VALUES.put("minecraft:grass_block", GRASS);
-        EMC_VALUES.put("minecraft:moss_block", GRASS);
-        EMC_VALUES.put("minecraft:raw_iron", IRON - 10); // a bit less (10) than the ingot
-        EMC_VALUES.put("minecraft:raw_gold", GOLD - 10); // a bit less (10) than the ingot
-        EMC_VALUES.put("minecraft:raw_copper", COPPER - 10); // a bit less (10) than the ingot
-        EMC_VALUES.put("minecraft:deepslate", DEEPSLATE);
-        EMC_VALUES.put("minecraft:cobbled_deepslate", COBBLESTONE); // has to be same as cobblestone
-        EMC_VALUES.put("minecraft:blackstone", COBBLESTONE); // has to be same as cobblestone
-        EMC_VALUES.put("minecraft:gilded_blackstone", GOLD); // should be same as gold
-        EMC_VALUES.put("minecraft:tuff", DEEPSLATE);
-        EMC_VALUES.put("minecraft:mud", DIRT);
-        EMC_VALUES.put("minecraft:clay_ball", LEAVES);
-        EMC_VALUES.put("minecraft:short_grass", DIRT);
-        EMC_VALUES.put("minecraft:tall_grass", DIRT);
-        EMC_VALUES.put("minecraft:fern", GRASS);
-        EMC_VALUES.put("minecraft:large_fern", GRASS);
-        EMC_VALUES.put("minecraft:azalea", LEAVES);
-        EMC_VALUES.put("minecraft:mycelium", LEAVES);
-        EMC_VALUES.put("minecraft:podzol", GRASS);
-        EMC_VALUES.put("minecraft:ice", GRASS);
-        EMC_VALUES.put("minecraft:vine", GRASS); // should be same as moss block!
-        EMC_VALUES.put("minecraft:glow_lichen", GRASS);
-        EMC_VALUES.put("minecraft:lily_pad", GRASS);
-        EMC_VALUES.put("minecraft:big_dripleaf", GRASS + 2);
-        EMC_VALUES.put("minecraft:small_dripleaf", GRASS + 2);
-        EMC_VALUES.put("minecraft:mangrove_roots", LEAVES);
-        EMC_VALUES.put("minecraft:sugar_cane", LEAVES * 5); // honey bottle / 3
-        EMC_VALUES.put("minecraft:pointed_dripstone", LEAVES);
-        EMC_VALUES.put("minecraft:hanging_roots", DIRT);
-        EMC_VALUES.put("minecraft:rooted_dirt", LEAVES);
-        EMC_VALUES.put("minecraft:cactus", LEAVES);
-        EMC_VALUES.put("minecraft:dead_bush", DIRT);
-        EMC_VALUES.put("minecraft:snowball", DIRT);
-        EMC_VALUES.put("minecraft:stripped_bamboo_block", BAMBOO * 9); // should be the same as bamboo_block
-        EMC_VALUES.put("minecraft:pitcher_pod", GRASS);
-        EMC_VALUES.put("minecraft:torchflower_seeds", GRASS);
-        EMC_VALUES.put("minecraft:honeycomb", LEAVES * 10); // this should not be more than 3, because of copper waxing: ((({copper_block_emc}[[+3]])*3+{redstone+blaze_rod})/4)[[+3]]
-        EMC_VALUES.put("minecraft:honey_bottle", LEAVES * 15); // sugar cane * 3
-        EMC_VALUES.put("minecraft:bee_nest", 174); // same as beehive
-        EMC_VALUES.put("minecraft:goat_horn", 1200);
-        EMC_VALUES.put("minecraft:frogspawn", 60);
-        EMC_VALUES.put("minecraft:turtle_egg", 70);
-        EMC_VALUES.put("minecraft:sniffer_egg", 80);
+        setEMCUnchecked("minecraft:cobblestone", COBBLESTONE);
```

Comment:
these arent strickly needed as this is the initial load, but still do the refactoring just to make it obvious that this is intentional that these puts get direct access to the underlying datastore.